src/orchard/inspect.clj

```diff
@@ -508,19 +539,25 @@
         (unindent))
     inspector))
 
-(defn- nav-datafy-tx [obj remove-nil-valued-entries?]
+(defn- nav-datafy-tx [remove-nil-valued-entries?]
   (keep (fn [[k v]]
-          (or (some->> (nav obj k v)
```

Comment:
Note the deletion of `nav`, explained in the comment below.

I'd say it was a bug - deftests were showing that we were performing `nav` over the same items repeatedly, which can be expensive in the face of side-effects.